draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -247,19 +252,19 @@ Application Service Providers' verifiers MAY wish to use dedicated DNS resolvers
+Delegated domain control validation lets a User delegate the domain control validation process for their domain to an Intermediary without granting the Intermediary the ability to make changes to their domain or zone configuration.  It is a variation of TXT record validation ({{txt-record}}) that inserts a CNAME record prior to the TXT record.
 
-The Intermediary gives the User a CNAME record to add for the domain and Application Service Provider being validated that points to the Intermediary's domain, where the actual validation TXT record is placed. The record name and base16-encoded (or base32-encoded) random tokens are generated as in {{random-token}}. For example:
+The Intermediary gives the User a CNAME record to add for the domain and Application Service Provider being validated that points to the Intermediary's domain, where the actual validation TXT record is placed.  The Intermediary's domain MUST be unique to the User, domain, and Application Service Provider. For example, if the User is Example Inc, then the CNAME record could be:
 
-    _service-challenge.example.com.  IN   CNAME  <intermediary-random-token>.dcv.intermediary.example.
+    _service-challenge.example.com.  IN   CNAME  example.com.service.example-inc.dcv.intermediary.example.
 
 The Intermediary then adds the actual Validation Record in a domain they control:
 
-    <intermediary-random-token>.dcv.intermediary.example.  IN   TXT "<provider-random-token>"
+    example.com.service.example-inc.dcv.intermediary.example.  IN   TXT  "<provider-random-token>"
 
 Such a setup is especially useful when the Application Service Provider wants to periodically re-issue the challenge with a new provider random token. CNAMEs allow automating the renewal process by letting the Intermediary place the random token in their DNS zone instead of needing continuous write access to the User's DNS.
 
-Importantly, the CNAME record target also contains a random token issued by the Intermediary to the User (preferably over a secure channel) which proves to the Intermediary that example.com is controlled by the User. The Intermediary must keep an association of Users and domain names to the associated Intermediary-random-tokens. Without a linkage validated by the Intermediary during provisioning and renewal there is the risk that an attacker could leverage a "dangling CNAME" to perform a "subdomain takeover" attack ({{SUBDOMAIN-TAKEOVER}}).
+During provisioning and renewal, or whenever ownership of a domain changes, there is the risk that an attacker could leverage a "dangling CNAME" to perform a "subdomain takeover" attack ({{SUBDOMAIN-TAKEOVER}}).  Using a subdomain tied to the User prevents these attacks.  This can be accomplished by choosing a name that includes the User's account name (as shown above) or another unique token linked to the User (as in {{random-token}}).
```

Comment:
(continuing the theme from above) I would prefer to not suggest using a name that includes the User's account name, since that is essentially some information disclosure that does not serve much purpose.  It maybe gives a little nudge to the provider to perform the User-account binding of the token but there's not any actual in-protocol mechanism to enforce that the provider does the binding, so I think our guidance here would be approximately as strong of a mechanism to get that binding.